CHANGES/994.feature

```diff
@@ -0,0 +1,8 @@
+Expanded on options available to `rpm content -t package upload`.
+
+The user can now:
+  * Upload an entire directory of RPMs
+  * Choose to have them added to their desired destionation as a
+    single new repository-version.
+  * Choose to have a new Publication created at the end of the
+    uploads.
```

Comment:
```suggestion
The user can now:
  * Upload an entire directory of RPMs
  * Choose to have them added to their desired destination as a single new repository-version.
  * Choose to have a new Publication created at the end of the uploads.
```

Reply:
Oh why....
Note: apart from the obvious unnecessary linebreaks, there was a typo in `destiOnation`.